utils/src/stake_table.rs

```diff
@@ -15,9 +19,64 @@ pub struct PermissionedStakeTableConfig {
     /// The list of public keys that are initially inserted into the
     /// permissioned stake table contract.
     #[serde(default)]
+    // Custom deserialization to handle toml file where the "stake" field is not provided.
+    // Defaults the "stake" field to 1 if not specified.
+    #[serde(deserialize_with = "deserialize_peer_config_keys")]
     pub public_keys: Vec<PeerConfigKeys<BLSPubKey>>,
 }
 
+fn deserialize_peer_config_keys<'de, D>(
```

Comment:
@imabdulbasit I think I prefer to add the dummy stake amount to the toml file (which the current orchestrator config also requires anyway) instead of adding the custom deserialization code just for this.

Reply:
SGTM! I'll remove it

Reply:
https://github.com/EspressoSystems/espresso-sequencer/pull/2377/commits/09ea7960d3fd6235b5114478b2fbc093aaaf868b